tests/e2e/default/wdio.conf.js

```diff
@@ -44,9 +43,44 @@ const baseConfig = {
   // then the current working directory is where your `package.json` resides, so `wdio`
   // will be called from there.
   //
-  specs: [
-    '**/*.wdio-spec.js',
-  ],
+  
+  suites: {
+    all: [
+      '**/*.wdio-spec.js',
+    ],
+    basic: [
+      './admin/**/*.wdio-spec.js',
+      './login/**/*.wdio-spec.js',
+      './translations/**/*.wdio-spec.js',
+      './more-options-menu/**/*.wdio-spec.js',
+      './users/**/*.wdio-spec.js'
+    ],
+    passive: [
+      './about/**/*.wdio-spec.js',
+      './navigation/**/*.wdio-spec.js',
+      './privacy-policy/**/*.wdio-spec.js',
+    ],
+    workflows: [
+      './analytics/**/*.wdio-spec.js',
+      './contacts/**/*.wdio-spec.js',
+      './reports/**/*.wdio-spec.js',
+      './targets/**/*.wdio-spec.js',
+      './tasks/**/*.wdio-spec.js',
+      './sms/**/*.wdio-spec.js',
+    ],
+    technical:[
```

Comment:
I wanted to suggest another name for this suite, because all suites are technical. 
I believe your choice of name wants to suggest that this is the low-level internal suite. 
What do you think about these classifications? "low-level" and "high-level" instead of `basic` + `passive` + `technical` ?